tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2510,3 +2510,33 @@ def test_limit_number_of_fields(self):
                     response.data["detail"]
                     == "You can view up to 20 fields at a time. Please delete some and try again."
                 )
+
+    def test_measurements_query(self):
+        data = load_data("transaction")
+        data["measurements"] = {"fp": {"value": 0.4}}
+        self.store_event(data, self.project.id)
+        query = {"field": ["measurements.fp", "p50()"]}
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
+        assert len(response.data["data"]) == 1
+        assert response.data["data"][0]["measurements.fp"] == 0.4
+
+    def test_measurements_aggregations(self):
+        data = load_data("transaction")
+        data["measurements"] = {"fp": {"value": 0.5}}
```

Comment:
```suggestion
        data["measurements"] = {"fcp": {"value": 0.5}}
```

I suggested a typo by accident 😬